src/File.ts

```diff
@@ -125,10 +132,20 @@ const tryRepetitive = async ({
         }
 
         const line = fileLines[listItemCache.position.start.line];
-
         if (line.includes(globalFilter)) {
             if (sectionIndex === originalTask.sectionIndex) {
-                listItem = listItemCache;
+                const dateFromFileName = new Lazy(() => DateFallback.fromPath(originalTask.path));
+                const taskFromLine = Task.fromLine({
+                    line,
+                    path: originalTask.path,
+                    precedingHeader: originalTask.precedingHeader,
+                    sectionStart: originalTask.sectionStart,
+                    sectionIndex: originalTask.sectionIndex,
+                    fallbackDate: dateFromFileName.value
+                });
+                if (taskFromLine?.identicalTo(originalTask) === true) {
+                    listItem = listItemCache;
+                }
```

Comment:
I see from the video that it falls through to `'Tasks: could not find task to toggle in the file.'` - but that is a bit too general for anyone to find the source of the problem...

Might it be possible to have an informative console log message if the task is not written.
It may help users that notice, and it will help track down the types of issues that are occurring.

Off the top of my head, perhaps including:

- The file name
- The expected raw task line
- The found raw task line
- Line numbers if known